Packages/com.chisel.graph/Chisel/ChiselGraphInstance.cs

```diff
@@ -0,0 +1,131 @@
+using Chisel.Core;
+using System;
+using System.Collections.Generic;
+using System.Reflection;
+using Unity.Collections;
+using Unity.Jobs;
+using UnityEngine;
+
+namespace Chisel.Nodes
+{
+    [ExecuteInEditMode]
+    public class ChiselGraphInstance : MonoBehaviour
+    {
+        public ChiselGraph graph;
+        public MeshFilter meshFilter;
+        public MeshRenderer meshRenderer;
+        public bool IsDirty { get; set; }
+
+        public List<GraphProperty> properties;
+        public Dictionary<string, GraphProperty> overriddenProperties;
+
+        CSGTree tree;
+        List<Mesh> meshes;
```

Comment:
can be
readonly List<Mesh> meshes = new List<Mesh>()
and you can remove the "meshes = new List<Mesh>();" in Start below

Reply:
same goes for properties/overriddenProperties